web/src/cypress/integration/schedules/temporarySchedule.ts

```diff
@@ -0,0 +1,284 @@
+import { Chance } from 'chance'
+import { testScreen } from '../../support'
+import { Schedule, User } from '../../../schema'
+import { DateTime, Interval } from 'luxon'
+import { round } from 'lodash-es'
+
+const c = new Chance()
+const dtFmt = "yyyy-MM-dd'T'HH:mm"
+
+function makeIntervalDates(): [string, string, number] {
+  const now = DateTime.local()
+  // year is either between now and 3 years in the future
+  const year = c.integer({
+    min: now.year,
+    max: now.year + c.integer({ min: 1, max: 3 }),
+  })
+  // random month, keep in the future if using current year
+  const month = c.integer({
+    min: year === now.year ? now.month : 1,
+    max: 12,
+  })
+  // random day, keep in future if using current month
+  const day = c.integer({
+    min: month === now.month ? now.day : 1,
```

Comment:
That's fair, this would probably "break" if it picked a year in the future but was the same month as ours, but the test would still pass since it's a future date.